committers.md

```diff
@@ -86,3 +86,45 @@ non-voting project maintainers.
     {% endif %}
   {% endfor %}
 </tbody></table>
+
+### **Becoming a committer**
+
+There are many ways to [contribute](https://arrow.apache.org/docs/developers/contributing.html)
+to the Apache Arrow project, including issue reports,
+documentation, tests, and code. Contributors with sustained, high-quality activity
+may be invited to become a committer by the PMC.
+
+Becoming a committer is a recognition of sustained
```

Comment:
This sounds redundant to the previous sentence.

Reply:
I tried to reduce the redundancy in fb77e71db04